providers/implementations/signature/eddsa_sig.c

```diff
@@ -341,9 +603,13 @@ const OSSL_DISPATCH ossl_ed25519_signature_functions[] = {
     { OSSL_FUNC_SIGNATURE_GET_CTX_PARAMS, (void (*)(void))eddsa_get_ctx_params },
     { OSSL_FUNC_SIGNATURE_GETTABLE_CTX_PARAMS,
       (void (*)(void))eddsa_gettable_ctx_params },
+    { OSSL_FUNC_SIGNATURE_SET_CTX_PARAMS, (void (*)(void))eddsa_set_ctx_params },
+    { OSSL_FUNC_SIGNATURE_SETTABLE_CTX_PARAMS,
+      (void (*)(void))eddsa_settable_ctx_params },
     { 0, NULL }
 };
 
+/* TODO: add streaming APIs for Ed448ph */
```

Comment:
Take the todo comment out.. We end up with lots of TODO's everywhere.. Create a issue if you need a reminder.

Reply:
okay.  I will remove the "TODO" comments.